code/modules/admin/verbs/freeze.dm

```diff
@@ -87,7 +87,7 @@ var/global/list/frozen_mob_list = list()
 
 //////////////////////////Freeze Mech
 
-/client/proc/freezemecha(var/obj/mecha/O as obj in GLOB.mechas_list)
+/client/proc/freezemecha(obj/mecha/O in GLOB.mechas_list)
```

Comment:
maybe make this /obj/mecha/M/admin_Freeze and have a counter part of it. Then declare a proc for these on atom level  which will be called by the freeze verb admins have